### PR TITLE
Expand documentation

### LINKED ISSUE #73: Add detailed instructions
- add instructions how to setup without docker images
   - add note that lb needs to be on public server while node and daemon can be behind router (private network)
- update docker-compose

### PR DESCRIPTION
**Pull request by MakMuftic**
**Expand README file with instructions for running Vedran loadbalancer**

### PR Checklist

- [x] I have run linter localy
- [x] I have run unit and integration tests locally
- [x] Rebased to master branch / merged master
- [x] Updated CHANGELOG.md

### Issues

Closes #73

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,38 +35,49 @@ go get github.com/NodeFactoryIo/vedran
...
 
-Load balancer is started using command `start`.
+First download latest prebuilt binaries [from releases](https://github.com/NodeFactoryIo/vedran/releases).
 
-```
-$ vedran -h
+Load balancer is started by invoking **start** command.
 
-vedran is a command line interface for polkadot load balancer
+For example `vedran-linux-amd64 start --auth-secret=supersecret`.
```
Comment (mpetrunic) [inline README.md]: we should probably archive binaries, so when extracted, they should all be called **vedran**

File: README.md [hunk 2]
```diff
@@ -35,38 +35,49 @@ go get github.com/NodeFactoryIo/vedran
...
 
-## Usage
+## Starting loadbalancer
 
-Load balancer is started using command `start`.
+First download latest prebuilt binaries [from releases](https://github.com/NodeFactoryIo/vedran/releases).
 
-```
-$ vedran -h
+Load balancer is started by invoking **start** command.
```
Comment (mpetrunic) [inline README.md:42]: add note that for more commands and options run `vedran --help` or `vedran start --help`

File: README.md (hunk 1 above)
Comment (mpetrunic) [reply README.md]:
```suggestion
For example `./vedran-linux-amd64 start --auth-secret=supersecret`.
```

Comment (MakMuftic) [reply README.md]: :+1:

File: README.md (hunk 2 above)
Comment (MakMuftic) [reply README.md:42]: :+1:

**Review by mpetrun5 — APPROVED** [review]

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**